README.md

```diff
@@ -13,11 +13,11 @@ check out our [PythonAnywhere demo site](http://defectdojo.pythonanywhere.com/).
 
 You can log in as an administrator like so:
 
-![Admin](https://inventropy.us/dd/admin.png) with password **administrator**
+![Admin](https://raw.githubusercontent.com/OWASP/django-DefectDojo/master/doc/img/admin.png)
```

Comment:
it would be preferred to at least clean up the images, so the username and password is not on the same line, but this is not required :)